src/Compilers/Core/CodeAnalysisTest/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -1777,7 +1763,7 @@ public override void Visit(IOperation operation)
                     }
                     if (operation.Kind == OperationKind.Argument)
                     {
-                        if (((IArgument)operation).ArgumentKind == ArgumentKind.ParamArray)
+                        if (((IArgument)operation).Parameter.IsParams)
```

Comment:
For purposes of this test, it doesn't matter whether or not the arguments were originally in array form or in expanded form.